### DISCUSSION TITLE
status "paid" per una fattura

### DISCUSSION THREAD

**Discussion by s-tools** [fattureincloud/api#324] [Q&A] [answered]
Sono i primi tentativi di integrazione del nostro gestionale con fatture in cloud.
Non riesco a trovare il modo di registrare un pagamento ad una fattura dopo aver inviato già la fattura.
Ho provato con  il comando  **CreateCashbookEntry** e mettendo in **Kind** il valore  **CashbookEntryKind.IssuedDocument**, ma l'id del documento non riesco a capire dove inserirlo, ho provato a scrivere
**Document = new CashbookEntryDocument()
	{
		Id = documentId,
		Type= "S.DO"
	},**
ma non viene collegato il pagamento alla fattura.
Mi potete aiutare?
Grazie

**Comment by valmoz** [comment]
Buongiorno @s-tools 
le consiglio prima di tutto di leggere la seguente guida, se non già fatto: [Invoice Creation](https://developers.fattureincloud.it/docs/guides/invoice-creation)
Cosa intende per "aver già inviato la fattura"?

Grazie
Mauro

**Reply by s-tools** [reply]
Buongiorno,
ho visto che in fase di creazione o modifica di una fattura possiamo inserire la data del pagamento della fattura, ma io ho la necessità di poter inserire il pagamento in un secondo momento e magari in modo differente da quello scritto in fattura.
Con "aver già inviato la fattura", intendevo che la fattura è già stata inviata come Fattura elettronica allo SDI
Grazie
Simone

**Comment by emajo** [comment, marked as answer]
Ciao @s-tools, i pagamenti della fattura possono essere modificati anche dopo l'invio all'SDI tramite l'apposita chiamata [Modify Issued Document](https://developers.fattureincloud.it/api-reference#put-/c/-company_id-/issued_documents/-document_id-).

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @s-tools 
abbiamo risposto alle sue domande?

Grazie
Mauro

**Reply by s-tools** [reply]
Buongiorno, 
scusate se non vi ho ringraziato prima ma ero impegnato a finire l'integrazione per il cliente.
Tutto a posto, grazie mille.
